common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -143,16 +142,11 @@ def B(*args, **kwargs):
     AUTH_ENTRY_LOGIN_2: '/account/login/',
     AUTH_ENTRY_REGISTER_2: '/account/register/',
 
-    # If linking/unlinking an account from the new student profile
-    # page, redirect to the profile page.  Only used if
-    # `FEATURES['ENABLE_NEW_DASHBOARD']` is true.
-    AUTH_ENTRY_PROFILE: '/profile/',
```

Comment:
The profile page we developed had the ability to link/unlink the user's account from a third-party account.  This is one entry point into the third-party auth pipeline, which behaves differently based on where the user entered the flow.

This should be safe to remove.  Even if someone had turned it on in production, the auth-entry value in the user's session gets overwritten whenever they enter auth pipeline.  Even if someone had enabled the new dashboard and started linking an account on the new profile page without completing the process (which seems unlikely), they could always go back to the login page and re-enter the auth flow.